src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import CalculatorUi from './components/Calculator';
 
 function App() {
```

Comment:
Great job with the project so far. However, one of the requirements of the project is that the `App` component should be a class-based component. Kindly update this component accordingly. ⬇️
![image](https://user-images.githubusercontent.com/33268927/211549360-dcab4aef-7dac-472c-93d8-635d804ba99c.png)